### PR TITLE
Fix IVR prompt response

### PR DESCRIPTION
**Pull request by Satendra-SR**
### Changes include:

- Fixes related to prompt details.
- Added Keypress to ivr prompt response table.

### CODE REVIEW

File: api/services/registration_service.py [hunk 1]
```diff
@@ -17,18 +17,16 @@ def set_init_data(self, jsonData):
     def handle_registration(self, jsonData):
         try:
             self.set_init_data(jsonData)
-            self.register(jsonData)
+            flow_run_uuid = helpers.fetch_by_key('run_uuid', jsonData)
+            call_log = models.CallLog.query.get_by_flow_run_uuid(flow_run_uuid)
```
Comment (rudresh) [inline api/services/registration_service.py:21]: Seems like duplicate of line after the `if` condition. Please remove

File: api/services/registration_service.py [hunk 2]
```diff
@@ -17,18 +17,16 @@ def set_init_data(self, jsonData):
...
-
-    def update_registration_details(self, jsonData):
-        try:
-            self.set_init_data(jsonData)
-            registration_details = models.Registration.query.get_latest_by_phone(self.user_phone)
-            if registration_details:
-                self.update(registration_details.id, jsonData)
-        except IndexError:
-            print("Failed to update registeration data")
+            print("Failed to handle registeration")
```
Comment (rudresh) [inline api/services/registration_service.py:29]: Spell check `registeration` :)

**Review by rudresh — CHANGES_REQUESTED** [review]
minor feedback

File: api/services/registration_service.py (hunk 1 above)
Comment (Satendra-SR) [reply api/services/registration_service.py]: yea! removed duplicacy

File: api/services/registration_service.py (hunk 2 above)
Comment (Satendra-SR) [reply api/services/registration_service.py]: Corrected.

**Review by rudresh — APPROVED** [review]

**Pull request merged**